user_interaction/templates/user_interaction/home_users.html

```diff
@@ -0,0 +1,118 @@
+
+{% extends 'core/base.html' %}
+
+{% load to_extended_user %}
+
+{% block title %}
+    Squire - Homepage
+{% endblock title %}
+
+{% block og-title %}
+    Squire Homepage
+{% endblock og-title %}
+
+{% block content-frame-class %}
+    container-fluid
+{% endblock %}
+
+{% block content %}
+    <div class="m-auto py-5" style="max-width: 1250px;">
+        <div class="row">
+            <div class="col-12  text-center ">
+                <p class="lead display-4">
+                    Hello {{ member.first_name }}
+                </p>
+                <p>
+                    Squire has levelled up! Have a look around and if there is anything off please let us know!
+                </p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-8">
+                <div class="row">
+                    <div class="col-12">
+                        <h3>
+                            Activities in the next 7 days
+                        </h3>
+                    </div>
+                    {% for activity_moment in activities %}
+                        <div class="col-12 col-md-6 col-xl-4 p-1">
+                            {% include "activity_calendar/snippet_activity_moment.html" %}
```

Comment:
I feel like three of these in a single row is a little tight; the blocks get rather thin and high